src/redirects.js

```diff
@@ -128,6 +127,58 @@ export const setup = (app) => {
 
 };
 
+/**
+ * Checks if the provided originalUrl matches are any of our datasetRedirectPatterns
+ * If the there is match, then return the new URL for the dataset.
+ * If there is not match, then return the original URL.
+ *
+ * @param {string} originalUrl
```

Comment:
This docstring should be changed to indicate that the input must be the pathname only (as the calling code does at the moment) - the function will return or destroy queries depending on if there's a `:version` match.

Reply:
Updated docstring in https://github.com/nextstrain/nextstrain.org/pull/812/commits/989b393ad1bca02ef53db6e420076fa801824911

(I also updated the param to `originalUrlPathname` to be clearer)

Reply:
👍